src/main/java/com/google/cloud/tools/app/config/module/ListConfiguration.java

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright 2016 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.tools.app.config.module;
+
+import com.google.cloud.tools.app.Option;
+
+import java.util.Collection;
+import java.util.Map;
+
+/**
+ * Configuration for {@link com.google.cloud.tools.app.module.ListAction}.
+ */
+public interface ListConfiguration {
+
+  /**
+   * Returns the modules collection to add to the command.
+   */
+  Collection<String> getModules();
+
+  /**
+   * Returns a map of Option, String pairs representing optional parameters.
+   */
+  Map<Option, String> getOptionalParameters();
```

Comment:
These methods are for the library's consumption. As an implementation detail, I thought it's easier if the configuration gives me a map of parameters I can just iterate and add to the command, instead of having to go through each one.

Which case do you see where having getters for each parameter would help?